dgraph/cmd/zero/run.go

```diff
@@ -185,6 +185,10 @@ func run() {
 		LudicrousMode:     Zero.Conf.GetBool("ludicrous_mode"),
 	}
 
+	if opts.nodeId == 0 {
+		log.Fatalf("ERROR: Cannot idx flag cannot be 0. Please try again with idx as a positive integer")
```

Comment:
Can you reword the error message? @rahulgurnani

Reply:
Thanks for pointing it out!

Reply:
Fixed on master and cherry picked here